src/dist/engine.js

```diff
@@ -0,0 +1,691 @@
+"use strict";
```

Comment:
Where did this `src/dist` directory come from? `lib/` is supposed to be the path with JS output

Reply:
Ops sorry pushed before cleaning. 

Will clean it up soon.